src/main/java/com/codetaylor/mc/onslaught/ModOnslaught.java

```diff
@@ -4,43 +4,45 @@
 import com.codetaylor.mc.athenaeum.module.ModuleManager;
 import com.codetaylor.mc.onslaught.modules.onslaught.ModuleOnslaught;
 import com.codetaylor.mc.onslaught.modules.onslaught.lib.LogFormatter;
-import net.minecraft.creativetab.CreativeTabs;
-import net.minecraft.init.Blocks;
-import net.minecraft.item.ItemStack;
-import net.minecraftforge.fml.common.Mod;
-import net.minecraftforge.fml.common.event.*;
-import org.apache.logging.log4j.LogManager;
-
 import java.io.IOException;
 import java.util.HashSet;
 import java.util.Set;
 import java.util.logging.FileHandler;
 import java.util.logging.Level;
 import java.util.logging.Logger;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.init.Blocks;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLConstructionEvent;
+import net.minecraftforge.fml.common.event.FMLInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLLoadCompleteEvent;
+import net.minecraftforge.fml.common.event.FMLPostInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.event.FMLServerAboutToStartEvent;
+import net.minecraftforge.fml.common.event.FMLServerStartedEvent;
+import net.minecraftforge.fml.common.event.FMLServerStartingEvent;
+import net.minecraftforge.fml.common.event.FMLServerStoppedEvent;
+import net.minecraftforge.fml.common.event.FMLServerStoppingEvent;
+import org.apache.logging.log4j.LogManager;
 
-@Mod(
-    modid = ModOnslaught.MOD_ID,
-    useMetadata=true
-)
+@Mod(modid = ModOnslaught.MOD_ID, useMetadata = true)
 public class ModOnslaught {
 
   public static final String MOD_ID = "onslaught";
+  public static final CreativeTabs CREATIVE_TAB =
+      new CreativeTabs(MOD_ID) {
 
-  public static Logger LOG;
+        @Override
+        public ItemStack getTabIconItem() {
 
+          return new ItemStack(Blocks.STONE);
+        }
+      };
+  public static Logger LOG;
   @SuppressWarnings("unused")
   @Mod.Instance
   public static ModOnslaught INSTANCE;
-
-  public static final CreativeTabs CREATIVE_TAB = new CreativeTabs(MOD_ID) {
-
-    @Override
-    public ItemStack getTabIconItem() {
-
-      return new ItemStack(Blocks.STONE);
-    }
-  };
-
```

Comment:
So all static final variables declared together in the [same place](https://github.com/Rebirth-of-the-Night/onslaught/pull/20/files/b4bbf2da431cc6cb8825f33e17429a12ea959f74#diff-df7d2fd1817c569c24d09aa42edbf98a49ebce850f83223298fcbf0288f7f754R33-R42).

Keeping things together makes things easier to learn. If all the public static variables are in the same place reliable, you don't have to look through the whole file (in case they've been stuffed somewhere further in).